### ISSUE TITLE
[Selenium] Move pull request containing tests to another repo

### ISSUE THREAD

**Issue by magdalenaBartkowiakJowsa** [devonfw/devonfw-testing#190]
Pull requests from https://github.com/devonfw-forge/mrchecker-source/pulls no
#9, #10, #11, #12 needs to be moved to https://github.com/devonfw/devonfw-testing/pulls

**Comment by gabrielstar** [comment]
@magdalenaBartkowiakJowsa  - can you please elaborate what do you mean by "move" pull requests ?
Do you mean that tests should be removed from checker-source and placed in devonfw-testsing ? or they should be kept in both ?

**Comment by magdalenaBartkowiakJowsa** [comment]
@gabrielstar yes, tests should be removed from checker-source and placed in devonfw-testsing.
